### ISSUE TITLE
 UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'styles' of undefined

### ISSUE THREAD

**Issue by beliayeu** [tomoyukikashiro/html2amp#69]
**Problem**
* amp generation fails when input html contains a link with invalid href

**Steps to reproduce**

case 1.  href points to a resource that does not exist

```
const path = require('path')
const html2amp = require('html2amp')
const html = `<html>
<head>
<link rel="stylesheet" href="/not/found.css" />
</head>
<body>
<h1>html2amp input</h1>
</body>
</html>`
const options = { cwd: path.join(process.cwd())}
const amp = html2amp(html, options)
amp.then(value => console.log(value))
```

case 2.  href contains a query param and points to a resource that exists in a local file system

```
const path = require('path')
const html2amp = require('html2amp')
const html = `<html>
<head>
<link rel="stylesheet" href="/assets/css/test.css?x=1" />
</head>
<body>
<h1>html2amp input</h1>
</body>
</html>`
const options = { cwd: path.join(process.cwd())}
const amp = html2amp(html, options)
amp.then(value => console.log(value))
```

**Suggestions**
* it would be great to not break amp generation and instead of it notify the operator (write a warning) once href contains invalid url
* allow link href to have query parameters once href points to a resource that exists on the file system

**Comment by tomoyukikashiro** [comment]
Hi, @beliayeu 

Thank you for your issue let me think for this soon ! 👍

**Comment by tomoyukikashiro** [comment]
@beliayeu I fixed this issue 👍 plz use [v1.3.0](https://github.com/tomoyukikashiro/html2amp/releases/tag/v1.3.0)
